src/deepsparse/yolact/schemas.py

```diff
@@ -68,8 +69,31 @@ class YOLACTInputSchema(ComputerVisionSchema, Splittable):
         "`postprocess` step (optional)",
     )
 
+    @classmethod
+    def from_files(cls, files: List[str], **kwargs) -> "YOLACTInputSchema":
+        """
+        :param files: list of file paths to create YOLOInput from
+        :param kwargs: extra keyword args to pass to YOLOInput constructor
+        :return: YOLOInput constructed from files
+        """
+        if "images" in kwargs:
+            raise ValueError(
+                f"argument 'images' cannot be specified in {cls.__name__} when "
+                "constructing from file(s)"
+            )
+        files_numpy = [numpy.array(Image.open(file)) for file in files]
+        input_schema = cls(images=files_numpy, **kwargs)
```

Comment:
This part of the definition we get for free from the parent class, could we use the superpowers of `super()`?